src/run.f90

```diff
@@ -572,68 +596,13 @@ module ac_run
 logical :: WaterTableInProfile, StartMode, NoMoreCrop
 logical :: GlobalIrriECw ! for versions before 3.2 where EC of
                          ! irrigation water was not yet recorded
-
+! Version 7.2
+integer(int32) :: LastIrriDAP
 
 
 contains
 
 
-subroutine open_file(fhandle, filename, mode, iostat)
```

Comment:
Why are those removed? not used?

Reply:
moved due to solving circularity in the compilation